pkg/ccl/backupccl/backup_planning.go

```diff
@@ -708,9 +708,8 @@ func backupPlanHook(
 						" subdirectory will be removed in a future release. Users can create a full backup via `BACKUP ... "+
 						"INTO <collectionURI>`, or an incremental backup on the latest full backup in their "+
 						"collection via `BACKUP ... INTO LATEST IN <collectionURI>`"))
-
 			} else {
-				initialDetails.Destination.Subdir = endTime.GoTime().Format(DateBasedIntoFolderName)
+				initialDetails.Destination.Subdir = unresolvedFullSubdir
```

Comment:
maybe we should keep this as-is for backwards compatibility and instead add a new field along side it in Destination e.g. "bool existing" which we'd set to true in the above two cases but not this one. Then we switch on existing later?

Also, shouldn't the notice above this be removed? I thought we realized that explicit subdirectory is _fine_ as long as it exists?